src/EditorFeatures/Core/Implementation/IntelliSense/AsyncCompletion/CommitManager.cs

```diff
@@ -341,6 +341,11 @@ internal static bool SendEnterThroughToEditor(CompletionRules rules, RoslynCompl
                 case EnterKeyRule.Always:
                     return true;
                 case EnterKeyRule.AfterFullyTypedWord:
+                    if (textTypedSoFar.LastOrDefault() == '\n')
```

Comment:
does this work on windows?  If so, might be worth commenting that it's ok to do \n, and not ned to do \r\n.

Reply:
Thank you, @cyrusnajmabadi!
Yes, it work on Windows because textTypeSoFar is a concatenation of individual characters typed by user.
FYI: @amadeusw

---
In reply to: [301855920](https://github.com/dotnet/roslyn/pull/37095#discussion_r301855920) [](ancestors = 301855920)